core/src/test/java/org/pentaho/di/connections/ConnectionManagerTest.java

```diff
@@ -61,13 +72,56 @@ public class ConnectionManagerTest {
 
   private ConnectionManager connectionManager;
 
-  private MemoryMetaStore memoryMetaStore = new MemoryMetaStore();
+  private Bowl bowl;
+
+  private MemoryMetaStore memoryMetaStore;
+
+
+  @BeforeClass
+  public static void setupClass() throws Exception {
+    KettleClientEnvironment.init();
+    DefaultBowl defaultBowl = DefaultBowl.getInstance();
+    assertNotNull( defaultBowl );
+
+    // Make sure that the default bowl has a meta-store, as this is a requirement of the connection manager,
+    // and otherwise would affect the default connection manager.
+    IMetaStore defaultMemoryMetaStore = new MemoryMetaStore();
+    defaultBowl.setMetastoreSupplier( () -> defaultMemoryMetaStore );
```

Comment:
why is this a different MemoryMetastore than the class member variable?

Reply:
This is being done in a static method.
Cannot use the static meta store for instance methods because it would accumulate things in it, causing some of the methods to fail.